examples/complete-postgres/main.tf

```diff
@@ -101,7 +101,7 @@ module "db" {
   parameters = [
     {
       name  = "autovacuum"
-      value = true
+      value = 1
```

Comment:
`true` works but on each subsequent plan/apply, terraform wants to change this to `1` so just setting to `1` to avoid this